config/crd/patches/cainjection_in_calicoconfigs.yaml

```diff
@@ -0,0 +1,8 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
```

Comment:
Webhook files are not needed until we add one. Lets remove this and webhook_in_calicoconfigs.yaml

Reply:
Removed these 2 files.